kafka_scripts/sample_producer.py

```diff
@@ -0,0 +1,76 @@
+import random
```

Comment:
Add `#!/usr/bin/env python3` to beginning of this script so that it can be ran directly. Also make it executable with `chmod +x`

Reply:
Sorry, this wasn't a good way (as we discussed IRL).